test/react-native/rn.patch.app.js

```diff
@@ -26,6 +26,17 @@ Sentry.init({
   release: '${SENTRY_RELEASE}',
   dist: '${SENTRY_DIST}',
   dsn: 'https://1df17bd4e543fdb31351dee1768bb679@o447951.ingest.sentry.io/5428561',
+  debug: true,
+  _experiments: {
+    replaysSessionSampleRate: 1.0,
+    replaysOnErrorSampleRate: 1.0,
```

Comment:
Currently all the tests are executed in one application session.

Let's set only `replaysOnErrorSampleRate` as that will ensure 
- the buffer to session Replay conversion is tested 
- as well the buffered replay id is assigned to the error event